wemake_python_styleguide/visitors/ast/classes.py

```diff
@@ -407,12 +407,15 @@ def _check_method_order(self, node: ast.ClassDef) -> None:
                 return
 
     def _ideal_order(self, first: str) -> int:
-        if first == '__new__':
-            return 4  # highest priority
-        if first == '__init__':
-            return 3
+        base_methods_order = {
+            '__new__': 5,  # highest priority
+            '__init__': 4,
+            '__call__': 3,
+        }
         if access.is_protected(first):
             return 1
         if access.is_private(first):
             return 0  # lowest priority
+        if first in base_methods_order.keys():
```

Comment:
this `if`, however it's totally correct, can be avoided:

```python
return base_methods_order.get(first, 2)
```

Also some constant can be used instead of number `2` to increase readability

Reply:
👍